core/domain/topic_domain.py

```diff
@@ -1259,13 +1265,22 @@ def update_url_fragment(self, new_url_fragment):
         self.url_fragment = new_url_fragment
 
     def update_thumbnail_filename(self, new_thumbnail_filename):
-        """Updates the thumbnail filename of a topic object.
+        """Updates the thumbnail filename and file size of a topic object.
 
         Args:
             new_thumbnail_filename: str|None. The updated thumbnail filename
                 for the topic.
         """
         self.thumbnail_filename = new_thumbnail_filename
+        file_system_class = fs_services.get_entity_file_system_class()
+        fs = fs_domain.AbstractFileSystem(file_system_class(
+            feconf.ENTITY_TYPE_TOPIC, self.id))
+        filename_prefix = 'thumbnail'
```

Comment:
Same thing, try to make this a common method/constant, especially since you are using it so many times throughout the codebase.

Reply:
Done.